gpustack/client/generated_model_file_client.py

```diff
@@ -107,3 +107,12 @@ def update(self, id: int, model_update: ModelFileUpdate):
     def delete(self, id: int):
         response = self._client.get_httpx_client().delete(f"{self._url}/{id}")
         raise_if_response_error(response)
+
+    def get_instances(
+        self, id: int, params: Dict[str, Any] = None
+    ) -> ModelInstancesPublic:
+        response = self._client.get_httpx_client().get(
+            f"{self._url}/{id}/instances", params=params
+        )
+        raise_if_response_error(response)
+        return ModelInstancesPublic.model_validate(response.json())
```

Comment:
`generated_*_client.py` are created by `make generate`. Changes should start from `codegen/templates`.